diesel-wasm-sqlite/Cargo.toml

```diff
@@ -5,8 +5,9 @@ edition = "2021"
 
 [dependencies]
 # diesel = { git = "https://github.com/xmtp/diesel", branch = "insipx/wasm-backend", default-features = false, features = ["i-implement-a-third-party-backend-and-opt-into-breaking-changes"] }
-diesel = "2.2"
+diesel = { version = "2.2", features = ["i-implement-a-third-party-backend-and-opt-into-breaking-changes"] }
```

Comment:
This is their "here be dragons" feature I guess

Reply:
yup, very aptly named :) 

but pretty much any api useful for this crate is gated behind it